### PR TITLE
EX-83 & 92 Omeka-S data persistence and add-ons

### PR DESCRIPTION
**Pull request by nomadicoder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,87 @@
-# omeka-s-docker
-Find more details and an example docker-compose.yml file in the README on Docker Hub, https://hub.docker.com/r/giocomai/omeka-s-docker/
+# tul_omeka-s
+
+*DRAFT*
+
+# Mounted Volumes for Persistence
```
Comment (sensei100) [inline README.md:5]: @nomadicoder It might be nice to add this to the kubernetes documentation as well.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,87 @@
...
+		- Volume name: As defined above
+		- Mount Point:/var/www/html/modules
+		- Subpath in Volume: modules
+	- >> Add Mount (if needed)
+		- Volume name: As defined above
+		- Mount Point: /var/www/html/themes
+		- Subpath in Volume: themes
+	- >> Save (bottom of form)
+- Workload will update, Status: "updating workload"
+- Rancho Workload Dashboard
```
Comment (sensei100) [inline README.md]:
```suggestion
- Rancher Workload Dashboard
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,87 @@
...
+- Change owner and group of mounted volumes
+
+```
+$ chown -R www-data:www-data files modules themes
+$ exit
+```
+
+# Install Extensions
+
+1. In browser, visit the Rancho Workload Dashboard
```
Comment (sensei100) [inline README.md]:
```suggestion
1. In browser, visit the Rancher Workload Dashboard
```

**Review by sensei100 — CHANGES_REQUESTED** [review]
@nomadicoder Looks good, maybe just a couple of typos for the dashboard name?

File: README.md (hunk 2 above)
Comment (cdoyle-temple) [reply README.md]: El Ranchero!

Comment (nomadicoder) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (nomadicoder) [reply README.md]: Fixed

**Review by sensei100 — APPROVED** [review]

**Pull request merged**